mautic/mautic
# Translation for date/datetime contactfield token

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

IF we set different language in configuration, then we expect translation date/datetime output for contactfield token translate too. This PR added support



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Switch default language in configuration to FR
3. Create translations/overrides/fr/CoreBundle/messages.ini with content

```
mautic.core.date.monday="Lundi"
mautic.core.date.tuesday="Mardi"
mautic.core.date.wednesday="Mercredi"
mautic.core.date.thursday="Jeudi"
mautic.core.date.friday="Vendredi"
mautic.core.date.saturday="Samedi"
mautic.core.date.sunday="Dimanche"
mautic.core.date.mon="Lun"
mautic.core.date.tue="Ma"
mautic.core.date.wed="Mer"
mautic.core.date.thu="Jeu"
mautic.core.date.fri="Ven"
mautic.core.date.sat="Sam"
mautic.core.date.sun="Dim"
mautic.core.date.january="Janvier"
mautic.core.date.february="Février"
mautic.core.date.march="Mars"
mautic.core.date.april="Avril"
mautic.core.date.may="Mai"
mautic.core.date.june="Juin"
mautic.core.date.july="Juillet"
mautic.core.date.august="Août"
mautic.core.date.september="Septembre"
mautic.core.date.october="Octobre"
mautic.core.date.november="Novembre"
mautic.core.date.december="Décembre"
mautic.core.date.jan="Jan"
mautic.core.date.feb="Fév"
mautic.core.date.mar="Mar"
mautic.core.date.apr="Avr"
mautic.core.date.may="May"
mautic.core.date.jun="Juin"
mautic.core.date.jul="Juil"
mautic.core.date.aug="Août"
mautic.core.date.sep="Sept"
mautic.core.date.oct="Oct"
mautic.core.date.nov="Nov"
mautic.core.date.dec="Déc"

```

4. Clear cache
5. Create email with `{contactfiedl=last_active|datetime} `token
6. Send email
7. You should see FR date string in email
